dashboard/config/routes.ts

```diff
@@ -9,6 +9,27 @@
     path: `/applications`,
     component: './Application',
   },
+  {
+    name: 'capability',
+    icon: 'AppstoreAddOutlined',
+    path: '/capabilities',
+    routes: [
+      {
+        path: '/capabilities',
+        redirect: `/Capability/Workloads`,
+      },
+      {
+        name: 'workloads',
+        path: '/capabilities/workloads',
+        component: './Capability/Workloads'
+      },
+      {
+        name: 'operating',
```

Comment:
Still call this "traits"?
Don't introduce new concepts since users might be confused.

Reply:
@zzxwill According to ISSUE #740  content "operating" is used here, is it changed back to "traits"?

Reply:
@sunny0826 Let's take hongchao's advice. Thanks.

Reply:
fixed